src/staticFilesServer.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+const http = require('http');
+const fs = require('fs');
+
+function staticFilesServer() {
+  const server = http.createServer((request, response) => {
+    const path = request.url;
```

Comment:
you don't set any headers but it's necessary. Use `application/octet-stream` for random files, and `text/plain` for your error message.